src/helpers/db/index.js

```diff
@@ -28,7 +29,8 @@ let encryptionKeys = {}
 let transforms = {}
 
 const DEBOUNCE_MS = process.env.NODE_ENV === 'test' ? 1 : 500
-const save = debounce(saveToDisk, DEBOUNCE_MS)
+const save = debounce(persistData, DEBOUNCE_MS)
+const EXPERIMENTAL_LOCAL_STORAGE = getEnv('EXPERIMENTAL_LOCAL_STORAGE')
```

Comment:
this could would probably be better done at runtime than at boot time? what if the value change over time (check the box)